corgi/core/files.py

```diff
@@ -28,8 +29,20 @@ def file_name(self) -> str:
     def __init__(self, obj: TimeStampedModel) -> None:
         self.obj = obj  # Model instance to manifest (either Component or Product)
 
-    def render_content(self) -> str:
-        kwargs_for_template = {"obj": self.obj}
+    @staticmethod
+    def get_created_at():
+        dt = datetime.now()
+        return dt.strftime("%Y-%m-%dT%H:%M:00Z")
+
+    def render_content(self, created_at: str = "", document_uuid: str = "") -> str:
+        if not created_at:
+            created_at = self.get_created_at()
+        document_namespace = f"{self.obj.name.replace('/', '_')}-{self.obj.version}"  # type: ignore[attr-defined]
```

Comment:
This is something which SDEngine does today when it publishes component manifests. I'm just making sure that the documentNamespace value matches the filename as it will be published in SDEngine.